revue-d-elevage-et-de-medecine-veterinaire-des-pays-tropicaux.csl

```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="fr-FR">
+  <info>
+    <title>Revue d'élevage et de médecine vétérinaire des pays tropicaux (French)</title>
+    <title-short>Rev.Elev.Med.Vet.Pays.Trop.</title-short>
+    <id>http://www.zotero.org/styles/revue-d-elevage-et-de-medecine-veterinaire-des-pays-tropicaux</id>
+    <link href="http://www.zotero.org/styles/revue-d-elevage-et-de-medecine-veterinaire-des-pays-tropicaux" rel="self"/>
+    <link href="http://remvt.cirad.fr/gb/presentation/note.html" rel="documentation"/>
```

Comment:
@BoyerCirad, I forgot to ask: did you base this on another CSL style? If so, please let us know which one, so we can add a "template" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link).